### PR TITLE
More docs improvements

### PR DESCRIPTION
**Pull request by andrunko**
Includes one actual bugfix (to install missing images) + general improvements.

https://phabricator.endlessm.com/T25587

### CODE REVIEW

File: gnome-help/C/shell-introduction.page [hunk 1]
```diff
@@ -108,7 +108,7 @@ like "Application windows" by using if:when and if:else ? -->
   typing to search your applications, files, folders, and the web.</p>
 
   <p if:test="platform:endless">To access your windows and applications,
-  click the Endless OS logo button at the bottom left corner of the screen.</p>
+  click the <gui>Endless (logo) button</gui> at the bottom left corner of the screen.</p>
```
Comment (wjt) [inline gnome-help/C/shell-introduction.page:114]: The commit message says that this standardizes on the term "Endless button" but here we have "(logo)". Let's consistently use "Endless button".

**Review by wjt — CHANGES_REQUESTED** [review]
Please merge after addressing the one comment.

File: gnome-help/C/shell-introduction.page (hunk 1 above)
Comment (andrunko) [reply gnome-help/C/shell-introduction.page:114]: This one specifically was actually on purpose as I wanted one place to mention the "logo" to help identifying the button (hence why between parenthesis), but yeah probably best to just drop it. I will update.

Comment (andrunko) [reply gnome-help/C/shell-introduction.page]: Done.

**Pull request merged**